sdk/core/azure-core/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "sourceFilesToProcess": null,
```

Comment:
Why `null` here?

Reply:
Why not?

Reply:
Btw, if `sourceFilesToProcess` is not present or null, the ApiView tool collects all the files under the specified input directory and treats them as the input to the API. The problem with this is that there may be files in the input directory which should *not* be collected, which is why we have `sourceFilesToSkip`.